scanners/boostsecurityio/semgrep/module.yaml

```diff
@@ -1,7 +1,7 @@
 api_version: 1.0
 
 id: boostsecurityio/semgrep
-name: BoostSecurity semgrep
+name: Ssemgrep
```

Comment:
(IMPORTANT) There's an extra s here, `Semgrep` seems to be the right casing

Reply:
fixed